apps/admin/src/services/mutations/getModalMutationOptions.ts

```diff
@@ -18,7 +18,7 @@ export const getModalMutationOptions = <
       useModalStore().loading = false;
     },
     onError(error) {
-      addErrorToNotifications(error, NotificationCategory.Modal);
+      addErrorToNotifications(error, {category: NotificationCategory.Modal});
```

Comment:
category zou juist los blijven zodat je een onderscheid kan maken tussen iets wat de locatie bepaalt en arbitraire details zoals of het van de api komt. Dus je krijgt `category` en `tags`